identifiers.md

```diff
@@ -2,12 +2,31 @@
+
+|Identifier Name | Definition | When to Use |
+|----|----|----|
+| `eventID` | An identifier for the set of information associated with a dwc:Event (something that occurs at a place and time). May be a global unique identifier or an identifier specific to the data set. | Used to distinguish between events in your data, *not* associated with an occurrence, e.g. a quadrat sample, an ROV deployment, a trolling event within a cruise |
+| `occurrenceID` | An identifier for the dwc:Occurrence (as opposed to a particular digital record of the dwc:Occurrence). In the absence of a persistent global unique identifier, construct one from a combination of identifiers in the record that will most closely make the dwc:occurrenceID globally unique. | Used to distinguish between occurrence records, i.e. when the presence of a taxon is detected (or not for absence records). If the same individual is detected multiple times, each detection will have unique a `occurrenceID` |
+| `organismID` | An identifier for the dwc:Organism instance (as opposed to a particular digital record of the dwc:Organism). May be a globally unique identifier or an identifier specific to the data set. | Use to identify a *specific* organism, an individual, or a specific group of organisms (e.g. a specific pod of cetaceans, a specific shark) |
+| `taxonID` | An identifier for the set of dwc:Taxon information. May be a global unique identifier or an identifier specific to the data set. | Used to identify a specific taxonomic rank, not commonly used in OBIS because `scientificNameID` is prioritized |
+| `scientificNameID` | An identifier for the nomenclatural (not taxonomic) details of a scientific name. | Used to provide an identifier for the name provided to `scientificName`. For OBIS, WoRMS LSIDs are recommended for this field |
+| `recordNumber` |  An identifier given to the dwc:Occurrence at the time it was recorded. Often serves as a link between field notes and a dwc:Occurrence record, such as a specimen collector's number. | Typically used when the occurrence is associated with a collected specimen. Different from `occurrenceID` because it may not be globally unique, whereas `occurrenceID` must be unique |
+| `materialEntityID` | An identifier for a particular instance of a dwc:MaterialEntity. Intended to uniquely and persistently identify a particular dwc:MaterialEntity within some context. | Used to identfy a physical object (i.e. the MaterialEntity: any kind of physical sample, preserved specimen, fossil, specific DNA molecule, etc.), instead of a digital representation of the object. E.g. an identifer for a specific tissue sample within an organization or institution |
```

Comment:
I think it is not possible to publish with materialEntityID for Occurrence core via the IPT interface yet. It is not found in https://rs.gbif.org/core/dwc_occurrence_2022-02-02.xml. 

I noted the justification on this issue https://github.com/tdwg/dwc/issues/455
> This would not prevent the use of any existing identifiers of material things within Darwin Core. However, it would be understood that it would be prefereable to use this term in place of dwc:materialSampleID. This proposal does not commit to a definition of the relationship of this term to dwc:materialSampleID, which is expected to be the subject of future discussion.

My question here is, I am wondering if we should perhaps "hide" this row until people can start using this in IPT? The reasoning is - I don't know if OBIS has a preference between using materialEntityID or materialSampleID for interpretation later on.